src/Section1/style.js

```diff
@@ -1,45 +1,94 @@
-
-import styled from 'styled-components';
-import { ReactComponent as Img } from '../programming 2.svg';
-
-
+import styled from "styled-components";
+import { ReactComponent as Img } from "../programming 2.svg";
 
 export const Container = styled.section`
-
-width: 100%;
-height: 608px;
-background: #FFD66C;
+  position: relative;
+  width: 100%;
+  height: 597px;
+  background: #ffd66c;
+
+  @media (max-width: 768px) {
+    width: 100%;
+    height: 603px;
+  }
+
+  @media (max-width: 376px) {
+    width: 100%;
```

Comment:
[Duplication] Since the width didn't change in both media queries, you don't need to re-declare it.